src/System-Time/Semaphore.extension.st

```diff
@@ -0,0 +1,23 @@
+Extension { #name : 'Semaphore' }
+
+{ #category : '*System-Time' }
+Semaphore >> wait: aDuration [
```

Comment:
What is the point of deprecating these two methods? I understand general idea of improving the modularity of the time package but Duration is still here. So fluent human friendly API is defined together with the main required class - Duration. Why we can't keep it? 

Now without this nice API users will model the timeouts as primitive numbers (no alternatives from Pharo libraries, right?). 
It always leads to duplicating comments explaining the units everywhere and duplicating methods providing alternatives for other kind of units. 
The Duration was introduced to address these kind of problems generally. And in Pharo it does the work really well.

I suggest to rethink this decision and to keep the fluent API.